cli/internal/prune/prune.go

```diff
@@ -206,58 +207,63 @@ func (c *PruneCommand) Run(args []string) int {
-			c.logError(c.Config.Logger, "", fmt.Errorf("Failed to copy root package.json: %w", err))
+			c.logError(c.Config.Logger, "", fmt.Errorf("failed to copy root package.json: %w", err))
 			return 1
 		}
 
 		if err := fs.CopyFile("package.json", filepath.Join(pruneOptions.cwd, "out", "json", "package.json"), fs.DirPermissions); err != nil {
-			c.logError(c.Config.Logger, "", fmt.Errorf("Failed to copy root package.json: %w", err))
+			c.logError(c.Config.Logger, "", fmt.Errorf("failed to copy root package.json: %w", err))
 			return 1
 		}
 	} else {
 		if fs.FileExists(".gitignore") {
 			if err := fs.CopyFile(".gitignore", filepath.Join(pruneOptions.cwd, "out", ".gitignore"), fs.DirPermissions); err != nil {
-				c.logError(c.Config.Logger, "", fmt.Errorf("Failed to copy root .gitignore: %w", err))
+				c.logError(c.Config.Logger, "", fmt.Errorf("failed to copy root .gitignore: %w", err))
 				return 1
 			}
 		}
 		if err := fs.CopyFile("package.json", filepath.Join(pruneOptions.cwd, "out", "package.json"), fs.DirPermissions); err != nil {
-			c.logError(c.Config.Logger, "", fmt.Errorf("Failed to copy root package.json: %w", err))
+			c.logError(c.Config.Logger, "", fmt.Errorf("failed to copy root package.json: %w", err))
 			return 1
 		}
 	}
 
 	next, err := yaml.Marshal(lockfile)
 	if err != nil {
-		c.logError(c.Config.Logger, "", fmt.Errorf("Failed to materialize sub-lockfile. This can happen if your lockfile contains merge conflicts or is somehow corrupted. Please report this if it occurs: %w", err))
+		c.logError(c.Config.Logger, "", fmt.Errorf("failed to materialize sub-lockfile. This can happen if your lockfile contains merge conflicts or is somehow corrupted. Please report this if it occurs: %w", err))
 		return 1
 	}
 	err = ioutil.WriteFile(filepath.Join(pruneOptions.cwd, "out", "yarn.lock"), next, fs.DirPermissions)
 	if err != nil {
-		c.logError(c.Config.Logger, "", fmt.Errorf("Failed to write sub-lockfile: %w", err))
+		c.logError(c.Config.Logger, "", fmt.Errorf("failed to write sub-lockfile: %w", err))
 		return 1
 	}
 	// because of yarn being yarn, we need to inject lines in between each block of YAML to make it "valid" syml
 	f, err := os.Open(filepath.Join(filepath.Join(pruneOptions.cwd, "out", "yarn.lock")))
 	if err != nil {
-		c.logError(c.Config.Logger, "", fmt.Errorf("Failed to massage lockfile: %w", err))
+		c.logError(c.Config.Logger, "", fmt.Errorf("failed to massage lockfile: %w", err))
 	}
 	defer f.Close()
 
 	output, err := os.Create(filepath.Join(pruneOptions.cwd, "out", "yarn-tmp.lock"))
 	writer := bufio.NewWriter(output)
 	if err != nil {
-		c.logError(c.Config.Logger, "", fmt.Errorf("Failed create tempory lockfile: %w", err))
+		c.logError(c.Config.Logger, "", fmt.Errorf("failed create tempory lockfile: %w", err))
 	}
 	defer output.Close()
 
-	writer.WriteString("# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.\n# yarn lockfile v1\n\n")
+	if ctx.Backend.Name == "nodejs-yarn" {
```

Comment:
In the future, need to add e2e tests for prune. Issue made here: https://github.com/vercel/turborepo/issues/553